python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging_advanced.py

```diff
@@ -228,8 +228,9 @@
     y_valid = np.array(ref_func(X_valid))
 
 # %%
-validation = ot.MetaModelValidation(x_valid, y_valid, krigingMeta)
-validation.computePredictivityFactor()
+metamodelPredictions = krigingMeta(x_valid)
+validation = ot.MetaModelValidation(y_valid, metamodelPredictions)
+validation.computeR2Score()
 
 # %%
 graph = validation.drawValidation()
```

Comment:
Can you please add the following correction to this file ? Replace the line

# We have to run the opitmization algorithm again.

with

# We have to run the optimization algorithm again.

Reply:
Ok. Fixed at 197b25abb.